src/main/webapp/pluggable/dataset-analisys-results.xhtml

```diff
@@ -0,0 +1,94 @@
+            </p:dataTable>
+            
+        </h:panelGroup>
+    </ui:fragment>
+
+    <ui:fragment rendered="#{!datasetAnalysisTab.showAnalysisResults()}">
+        <div class="panel panel-default">
+            <div class="panel-body">
+                <div class="col-sm-9">
+                    <ui:fragment rendered="#{datasetAnalysisTab.isDatasetInDraft()}">
+                        <p>
+                            <h:outputText value="#{mxrdrBundle['dataset.analysisTab.dataset.in.draft.message']}" />
+                        </p>
+                    </ui:fragment>
+                    <ui:fragment rendered="#{datasetAnalysisTab.isDatasetUnderEmbargo() and !datasetAnalysisTab.isPermissionToViewFiles()}">
+                        <p>
+                            <h:outputFormat value="#{mxrdrBundle['dataset.analysisTab.embargo.message']}">
+                                <f:param value="#{datasetAnalysisTab.getEmbargoDateForDisplay()}"/>
+                            </h:outputFormat>
+                        </p>
+                    </ui:fragment>
+                    <ui:fragment rendered="#{!datasetAnalysisTab.isDatasetInDraft() and
+                                              (!datasetAnalysisTab.isDatasetUnderEmbargo() or
+                                              (datasetAnalysisTab.isDatasetUnderEmbargo() and datasetAnalysisTab.isPermissionToViewFiles()) and
+                                              datasetAnalysisTab.isAnalysisInProgress())}">
+                        <p>
+                            <h:outputText value="#{mxrdrBundle['dataset.analysisTab.analysis.in.progress.message']}" />
+                        </p>
+                    </ui:fragment>
+                    <ui:fragment rendered="#{!datasetAnalysisTab.isDatasetInDraft() and
+                                              (!datasetAnalysisTab.isDatasetUnderEmbargo() or
+                                              (datasetAnalysisTab.isDatasetUnderEmbargo() and datasetAnalysisTab.isPermissionToViewFiles()) and
+                                              datasetAnalysisTab.isAnalysisNotPerformed())}">
+                        <p>
+                            <h:outputText value="#{mxrdrBundle['dataset.analysisTab.analysis.not.performed.message']}" />
+                        </p>
+                    </ui:fragment>
+                    <ui:fragment rendered="#{!datasetAnalysisTab.isDatasetInDraft() and
+                                              (!datasetAnalysisTab.isDatasetUnderEmbargo() or
+                                              (datasetAnalysisTab.isDatasetUnderEmbargo() and datasetAnalysisTab.isPermissionToViewFiles()) and
+                                              datasetAnalysisTab.isAnalysisFailure())}">
+                        <p>
+                            <h:outputText value="#{mxrdrBundle['dataset.analysisTab.analysis.failed.message']}" />
+                        </p>
+                    </ui:fragment>
```

Comment:
In case of failure we still want to show artifacts

Reply:
Done